packages/core/src/components/footer/footer.tsx

```diff
@@ -17,10 +18,13 @@ export class TdsFooter {
   /** Mode variant of the component, based on current mode. */
   @Prop() modeVariant: 'primary' | 'secondary' = null;
 
+  const copyrightText = 'Copyright © ' + new Date().getFullYear() + ' Scania';
```

Comment:
use `Template literals`?

```suggestion
  const copyrightText = `Copyright © ${new Date().getFullYear()} Scania`;
```

Reply:
Good call ;)